src/Polymorph-Widgets/UITheme.class.st

```diff
@@ -1288,14 +1288,24 @@ UITheme >> configureWindowDropShadowFor: aWindow [
 UITheme >> configureWindowLabelAreaFor: aWindow [
```

Comment:
To avoid all the checks we could have a little hierarchy OSWindowLayout and subclasses. But we can do that in a second step.